scheduler/server/job_state.go

```diff
@@ -264,3 +270,38 @@ func (j *jobState) removeTaskFromStartTimeMap(jobID string, taskID string, start
 		delete(j.tasksByJobClassAndStartTimeSec, timeBucket)
 	}
 }
+
+func (j *jobState) logInconsistentStateValues() {
+	if j.TasksCompleted != len(j.Completed) {
+		log.Errorf("job:%s,%s,%s: TasksCompleted count (%d) != number entries in Completed map (%d)",
```

Comment:
suggest adding a magic string to the beginning of all these logs so it's easy to find any of them e.g. "inconsistent: "